axolotlsay-js/build.sh

```diff
@@ -4,59 +4,36 @@ set -eu
 
 EXE_BASENAME=axolotlsay-js
 
-install_bun_windows() {
-  powershell -c "irm bun.sh/install.ps1 | iex"
-}
+# Install devDependencies to get bun
+npm install --include=dev
 
-install_bun_unix() {
-  curl -fsSL https://bun.sh/install | bash
-}
-
-# Don't install bun in interactive shells; just assume we have it
-if [ -n "${CI:-}" ] && ! type bun >/dev/null; then
-  case "$(uname -s)" in
-    MINGW* | MSYS* | CYGWIN* | Windows_NT)
-      install_bun_windows
-      ;;
-    *)
-      install_bun_unix
-      ;;
-    esac
-
-  # So we don't have to reload the config it writes
-  # Note: this is also valid for Windows, which also places
-  # it in $HOME
-  export BUN_INSTALL="$HOME/.bun"
-  export PATH="$HOME/.bun/bin:$PATH"
-fi
-
-bun install
-
-args=
+bun_args=
 
 # If we're building for cargo-dist, specify which target to build and what the
 # filename should be. cargo-dist will expect one output with a known filename.
 if [ -n "${CARGO_DIST_TARGET:-}" ]; then
     case "${CARGO_DIST_TARGET}" in
         x86_64-pc-windows-msvc)
-            args="--target bun-windows-x64 --outfile ${EXE_BASENAME}.exe"
+            bun_args="--target bun-windows-x64 --outfile ${EXE_BASENAME}.exe"
             ;;
```

Comment:
can we add the obvious-to-predict windows-arm64 target here for future proofing? also is musl supported by bun?

(I also wonder if making this setting variables and just having one compile expr at the end would be nicer for maintenance but it's kinda potato potahtoh)

Reply:
As far as I can tell, musl hasn't yet launched official binaries on musl yet. That may come in the future. And yeah, good call on aarch64 Windows.

Reply:
Just checked, and bun isn't supported on arm64 Windows yet. I'll make the switch statement recognize it but output x86_64 binaries.